manifests/charts/istio-cni/templates/daemonset.yaml

```diff
@@ -55,10 +55,6 @@ spec:
 {{ toYaml .Values.podAnnotations | indent 8 }}
         {{- end }}
     spec:
-{{if .Values.ambient.enabled }}
```

Comment:
I am wondering if we want to keep this as an optional thing (permanently or for a few releases while we validate things)...
I cannot come up with many great reasons though

Reply:
I think we can keep a  "shareHostNetwork" option for some releases, I added it to the compatibility profiles as part of the deprecation process.
If someone comes up during the deprecation process with a good reason to keep it then we could retain it